newrelic/exporter.go

```diff
@@ -78,13 +79,18 @@ func (e *Exporter) ExportSpan(ctx context.Context, span *tracesdk.SpanData) {
 }
 
 // Export exports metrics to New Relic.
-func (e *Exporter) Export(_ context.Context, r *resource.Resource, cps metricsdk.CheckpointSet) error {
-	return cps.ForEach(func(record metricsdk.Record) error {
-		m, err := transform.Record(e.serviceName, r, record)
+func (e *Exporter) Export(_ context.Context, cps metricsdk.CheckpointSet) error {
+	return cps.ForEach(e, func(record metricsdk.Record) error {
+		m, err := transform.Record(e.serviceName, record.Resource(), record)
```

Comment:
Could probably just remove the `Resource` argument from the function signature as it will be included in the passed `record`.